### DISCUSSION TITLE
How to achieve Github Profile Badge ?

### DISCUSSION THREAD

**Discussion by Isuru-Eranda** [community/community#70827] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi 👋  everyone, I'm Isuru Eranda

![B](https://github.com/community/community/assets/132377805/e3b31cb5-ae13-4ae4-9bc3-000f91a4ce77)

**Comment by aalexander47** [comment]
hello @Isuru-Eranda thanks for your reply its was great help 
Here's what you can check `https://github.com/drknzz/GitHub-Achievements` it shows how you can achieve it or what all you outta do  for it

**Reply by Isuru-Eranda** [reply]
Thank you very much for your reply.

**Comment by MayodhaThathsarani** [comment, marked as answer]
Study this Github repository 
Link - https://github.com/Thinkright20/Profile-Badges

**Reply by Isuru-Eranda** [reply]
Thank you very much for your answer.

**Reply by mymarius** [reply]
wow

**Comment by mymarius** [comment]
good bro

**Comment by ahlaiptek** [comment]
nice

**Comment by dorochadev** [comment]
you should mark this reply as the answer so that i can get a badge ;)

**Comment by sarath-dbug** [comment]
Study this Github repository
Link - https://github.com/Thinkright20/Profile-Badges

**Comment by ghost** [comment]
you should mark this reply as the answer so that i can get a badge !

**Comment by Hazrat-Ali9** [comment]
Study this Github Repository
Link : https://github.com/Thinkright20/Profile-Badges

**Comment by deconasser** [comment]
Visit: https://github.com/Thinkright20/Profile-Badges or watch youtube video:  https://www.youtube.com/watch?v=oJWFfp3sGYA&t=619s

**Comment by 3bHussein** [comment]
thank you

**Comment by SahrulRD-Smatel** [comment]
Thank you very much
